srcJava/com/rti/perftest/ddsimpl/RTIPublisher.java

```diff
@@ -171,9 +171,9 @@ public long getPulledSampleCount()
         return status.pulled_sample_count;
     }
 
-    public void wait_for_acknowledgments(Duration_t timeout){
+    public void wait_for_acknowledgments(int sec, int nsec){
```

Comment:
Could you add a space between ")" and "{"

Reply:
Done